### DISCUSSION TITLE
Welcome to flutter_stripe_payment Discussions!

### DISCUSSION THREAD

**Discussion by jonasbark** [jonasbark/flutter_stripe_payment#231] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SAKSHAM-flutter** [comment]
this kind of error show com.stripe.android.exception.InvalidRequestException: As per Indian regulations, export transactions require a customer name and address. More info here: https://stripe.com/docs/india-exports;
.
.
.
but i am not understand where is pass name and address.


my fuction is here:-
static Future<StripeTransactionResponse> payViaExistingCard(
      {String amount, String currency, CreditCard card}) async {
    try {
      var paymentMethod = await StripePayment.createPaymentMethod(
          PaymentMethodRequest(
              card: card));
      var paymentIntent =
          await StripeService.createPaymentIntent(amount, currency);
      var response = await StripePayment.confirmPaymentIntent(PaymentIntent(
          clientSecret: paymentIntent['client_secret'],
          paymentMethodId: paymentMethod.id));
      if (response.status == 'succeeded') {
        return new StripeTransactionResponse(
          message: 'Transaction successful',
          success: true,
          id: paymentMethod.id.toString(),
        );
      } else {
        return new StripeTransactionResponse(
          message: 'Transaction failed',
          success: false,
          id: paymentMethod.id != null && paymentMethod.id != ""
              ? paymentMethod.id.toString()
              : "12345",
        );
      }
    } on PlatformException catch (err) {
      return StripeService.getPlatformExceptionErrorResult(err);
    } catch (err) {
      return new StripeTransactionResponse(
          message: 'Transaction failed: ${err.toString()}', success: false);
    }
  }

**Comment by SaferAc** [comment]
Could you include pay via oxxo Method https://stripe.com/docs/payments/oxxo/accept-a-payment?platform=web
